test/check-cli

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import signal
+import subprocess
+import sys
+
+# import Cockpit's machinery for test VMs and its browser test API
+sys.path.append(os.path.join(os.path.dirname(__file__), "../bots/machine"))
+import testvm
+
+
+def main(image):
+    network = testvm.VirtNetwork(0)
+    machine = testvm.VirtMachine(image, networking=network.host(), memory_mb=2048)
+    machine.start()
+    machine.wait_boot()
+
+    # run a command to force starting the SSH master
+    machine.execute('uptime')
+
+    ssh_command = ["ssh", "-o", "ControlPath=" + machine.ssh_master,
+                          "-p", machine.ssh_port,
+                          machine.ssh_user + "@" + machine.ssh_address]
+
+    print(f"Virtual machine running {image} is up. Connect to it via:")
+    print(" ".join(ssh_command))
+    print()
+
+    r = subprocess.run(ssh_command + ["CLI=/usr/bin/composer-cli", "/tests/test_cli.sh"])
```

Comment:
Add 3rd parameter the path to the test you want to execute. Probably remove the target from Makefile ?

Will require `./$TEST` to change to `$TEST` inside test_cli.sh to account for the fact that you are passing absolute path. I've tried it locally and it worked for me.

Reply:
I converted `test/check-cli` to use `unittest` and included all scripts under `tests/cli`. I've also changed `./$TEST` to `$TEST`. I left the `make` targets in there, because you said you sometimes manually run the tests for different rpms?

We probably want to run some of the tests in `/tests/composer` in virtual machines as well.